src/Resources/contao/templates/backend/be_pagination.html5

```diff
@@ -1,5 +1,6 @@
 
-<div class="pagination <?= $this->class ?>">
+<nav class="pagination <?= $this->class ?>" role="navigation" aria-label="<?= $GLOBALS['TL_LANG']['MSC']['paginationAriaLabel'] ?>">
+  <a href="<?= $this->request ?>#skipNavigation" class="invisible"><?= \StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['skipNavigation']); ?></a>
```

Comment:
This is not correct. First of all, you must not use a hard coded skip ID, because there might be more than one pagination menu and it would lead to ambiguous IDs. Second of all, you also have to add the target element: 

https://github.com/contao/core-bundle/blob/master/src/Resources/contao/templates/modules/mod_navigation.html5#L13

Reply:
@leofeyer Am i right, or is anything to do here?

Reply:
> or is anything to do here?

Of course there is:

> First of all, you must not use a hard coded skip ID, because there might be more than one pagination menu and it would lead to ambiguous IDs. Second of all, you also have to add the target element:

None of this has been fixed, has it?

Reply:
@leofeyer What about my comment?

> We are talking about the backend here, aren't we?

Reply:
There might be more than one pagination menu in the back end, too.

Reply:
@leofeyer Now, I've added the target element for the naviation skipping.

BUT: I'm still of the opinion, that multiple paginations in the back end are not possible. In the current implementation there is no unique identifier for a pagination. And scrolling will be done via URL parameter `lp` ... if there is more than one pagination no decision can be done, which list to scroll.